style.js

```diff
@@ -0,0 +1,256 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.navlink');
+// const span = document.getElementsByClassName('bar')[2];
```

Comment:
- [OPTIONAL] It will be nice to remove commented code from your code base as this will make your code base clean and less crowded.